.changelog/2513.feature.md

```diff
@@ -0,0 +1,9 @@
+Runtime node admission policies.
+
+Each runtime should be able to define its node admission policy.
```

Comment:
Update the text to reflect that this is what is being added not that it should be added :-)

Reply:
ok